enhancements/update/manifest-install-levels.md

```diff
@@ -0,0 +1,166 @@
+---
+title: manifest-install-levels
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+approvers:
+  - TBD
+creation-date: 2020-09-12
+last-updated: 2020-09-12
+status: implementable
+---
+
+# Manifest Install Levels
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+[Several operators][extra-manifests] pass manifests [like these][extra-manifests-example] to the cluster-version operator which are nice-to-have, [create-only][] defaults, but not critical to bootstrap completion.
+That's fine, and makes life in the post-install clusters easier.
+However, when a user feeds the installer their own alternative content, [cluster-bootstrap][] will [race][] the cluster-version operator to push the content into the cluster.
+Sometimes cluster-bootstrap wins, and the user-provided manifest ends up in-cluster.
+But sometimes the cluster-version operator wins, and the user-provided content is forgotten when the create-only default ends up in-cluster.
+This enhancement adds a new enhancement to delay non-critical manifests to later in the installation, ensuring the cluster-version operator will always lose the race and cluster-bootstrap will push the user-provided manifest into the cluster.
+
+## Motivation
+
+### Goals
+
+* Respect manifests that users feed the installer, even if they target a resource that is also backed by a cluster-version operator, create-only manifest.
+
+### Non-Goals
+
+* Completely ordering installation.
+    The cluster-version operator used to order manifests during installation, but we [dropped that][parallel-install] for quicker installs.
+    This enhancement should restore enough install-time ordering to avoid the races, but not so much that it significantly delays installs.
+
+## Proposal
+
+This enhancemnt defines a new manifest annotation, `release.openshift.io/install-level`, which allows manifests to be ordered by run-level during installation.
+The default value is `0`, and currently the only other allowed value is `1`.
+
+The cluster-version operator will ignore the new annotation for all phases except installation.
```

Comment:
Please describe what this means.

Reply:
I've explicitly mentioned all three CVO phases in 916cef8 -> 949a91b.  Hopefully it's more clear now that this new annotation is completely ignored outside of the install phase.